astropy/convolution/setup_package.py

```diff
@@ -0,0 +1,24 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import os
+from numpy import get_include as get_numpy_include
```

Comment:
Please check the other submodules how the numpy includes were added (either io.ascii or io.fits), and use the same logic here, that will at least fix the egg builds.

Reply:
Will do, thanks.